src/hyperfine/shell.rs

```diff
@@ -15,54 +14,91 @@ pub struct ExecuteResult {
 
     /// The exit status of the process
     pub status: ExitStatus,
+
+    /// Stdout
+    pub stdout: String,
```

Comment:
I would prefer this to be `Option<String>` (use `None` instead of empty string).